### PR TITLE
[custom-functions-runtime] Add missing functionName property

### PR DESCRIPTION
**Pull request by alison-mk**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: Fixes issue noted here https://github.com/OfficeDev/office-js-docs-reference/issues/1000
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/custom-functions-runtime/index.d.ts [hunk 1]
```diff
@@ -48,6 +48,11 @@ declare namespace CustomFunctions {
      * Provides information about the invocation of a custom function.
      */
     interface Invocation {
+        /**
+         * The name of this function.
+         * [Api set: CustomFunctionsRuntime 1.1]
+         */
+        functionName: string;
```
Comment (akrantz) [inline types/custom-functions-runtime/index.d.ts:55]: @alison-mk  Doesn't this need to be `functionName?: string` since it may be undefined.

**Review by akrantz — CHANGES_REQUESTED** [review]
See comment.

File: types/custom-functions-runtime/index.d.ts (hunk 1 above)
Comment (alison-mk) [reply types/custom-functions-runtime/index.d.ts:55]: Thanks for the review! Discussing on [internal work item](https://office.visualstudio.com/OC/_workitems/edit/5319473).

**Comment by alison-mk** [comment]
Thanks for the review @akrantz! I incorporated your feedback. Can you take another look?

**Review by akrantz — APPROVED** [review]

File: types/custom-functions-runtime/index.d.ts [hunk 2]
```diff
@@ -48,6 +48,11 @@ declare namespace CustomFunctions {
      * Provides information about the invocation of a custom function.
      */
     interface Invocation {
+        /**
+         * The name of this function.
+         * [Api set: CustomFunctionsRuntime 1.1]
```
Comment (akrantz) [inline types/custom-functions-runtime/index.d.ts:53]: @alison-mk Is this part of CustomFunctionsRuntime 1.1 or a later version?

**Review by akrantz — CHANGES_REQUESTED** [review]
Alison, please see my other comment about the requirement set version number.

File: types/custom-functions-runtime/index.d.ts (hunk 2 above)
Comment (alison-mk) [reply types/custom-functions-runtime/index.d.ts:53]: This is part of CustomFunctionsRuntime 1.1. It was previously undocumented. Noted by customer in invocation.functionName property OfficeDev/office-js-docs-reference#1000.

**Review by akrantz — APPROVED** [review]

**Pull request merged**